agent/05_agent_configure.sh

```diff
@@ -176,7 +176,7 @@ function get_mirror_info {
     tmpmirrorinfo=$(mktemp --tmpdir "mirror--XXXXXXXXXX")
     _tmpfiles="$_tmpfiles $tmpmirrorinfo"
     if [[ ${MIRROR_COMMAND} == "oc-adm" ]]; then
-       sed -n '/imageContentSources/,/^ *$/p' ${MIRROR_LOG_FILE} | tail -n+2 > ${tmpmirrorinfo}
+       sed -n -E '/imageContentSources|imageDigestSources/,/^ *$/p' ${MIRROR_LOG_FILE} | tail -n+2 > ${tmpmirrorinfo}
```

Comment:
From a maintenance point of view maybe it could be slightly more clear to have something like this, with a proper comment explaining why we're fetching one field instead of another:

```
if printf '4.14\n%s\n' "$(openshift_version)" | sort -V -C; then
...
```